examples/visualization/plot_channel_epochs_image.py

```diff
@@ -74,6 +74,6 @@ def order_func(times, data):
 plt_times = np.linspace(0, .2, len(epochs))
 
 plt.close('all')
-mne.viz.plot_epochs_image(epochs, [good_pick, bad_pick], sigma=0.5, vmin=-100,
-                          vmax=250, colorbar=True, order=order_func,
+mne.viz.plot_epochs_image(epochs, [good_pick, bad_pick], sigma=.5,
+                          cmap='magma', order=order_func,
```

Comment:
magma is inappropriate for symmetric colormaps. Please just remove passing a custom cmap